grpc-api/src/test/java/io/deephaven/grpc_api/runner/OperationsTest.java

```diff
@@ -0,0 +1,46 @@
+package io.deephaven.grpc_api.runner;
+
+import io.deephaven.grpc_api.util.ExportTicketHelper;
+import io.deephaven.proto.backplane.grpc.EmptyTableRequest;
+import io.deephaven.proto.backplane.grpc.ExportedTableCreationResponse;
+import io.deephaven.proto.backplane.grpc.Ticket;
+import io.deephaven.proto.backplane.grpc.TimeTableRequest;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OperationsTest extends DeephavenApiServerSingleAuthenticatedBase {
+
+    @Test
+    public void emptyTable() {
+        ExportedTableCreationResponse response = channel.tableBlocking()
+                .emptyTable(EmptyTableRequest.newBuilder().setResultId(id1()).setSize(123).build());
+        checkResponse(response, id1(), true, 123);
+    }
+
+    @Test
+    public void timeTable() {
+        ExportedTableCreationResponse response = channel.tableBlocking()
+                .timeTable(TimeTableRequest.newBuilder().setResultId(id1()).setPeriodNanos(TimeUnit.SECONDS.toNanos(1))
+                        .build());
+        checkResponse(response, id1(), false, 0);
+    }
+
+    // todo: exercise all operations
```

Comment:
Are you intending to do that within this PR, or should we toss up an issue to circle back? (Clearly this change set already has value.)